### DISCUSSION TITLE
关于多功能的问题

### DISCUSSION THREAD

**Discussion by TXIuTnVsbA** [XTLS/Xray-core#387] [Q&A] [answered]
我有一台香港服务器和一台美国服务器，想让xray同时拥有科学上网和流量中转的功能，这样的话，那台美国的服务器再也不会怕速度慢了。

我思路是这样的，监听俩个inbound和俩个outbound。

科学上网：
国内  ->  inbound_A(香港服务器) -> outbound_A(香港服务器) 

流量转发:
国内  ->  inbound_B(香港服务器) -> outbound_B(香港服务器) -> inbound(美国服务器) -> outbound(美国服务器)

请问，这个功能怎么搞？多配置就行吗？

**Comment by AkinoKaede** [comment, marked as answer]
用路由就行
